python/perspective/perspective/client/table_api.py

```diff
@@ -0,0 +1,147 @@
+################################################################################
+#
+# Copyright (c) 2019, the Perspective Authors.
+#
+# This file is part of the Perspective library, distributed under the terms of
+# the Apache License 2.0.  The full license can be found in the LICENSE file.
+#
+from random import random
+from functools import partial
+from .dispatch import async_queue, subscribe, unsubscribe
+from .view_api import view as make_view
+
+
+def table(client, data, index=None, limit=None, name=str(random())):
```

Comment:
Same, and also it's probably a good idea to just make `name` a required arg here as this `table` is private to the library.